### DISCUSSION TITLE
Hi there, well done!! is this support server mode? yep like monitor agent

### DISCUSSION THREAD

**Discussion by MorrisLu-Taipei** [GyulyVGC/sniffnet#569] [Ideas]
Server mode like..
I have 10 pc machine in my place, i would like to install Sniffnet Server in 1 machine, all others 9 machine can be install Sniffnet client (like monitor) in each. I can monitor all 10 server in just 1 machine

**Comment by GyulyVGC** [comment]
Hi @MorrisLu-Taipei, I'm happy you're finding Sniffnet helpful.
This feature is already planned, but not for the short term unluckily.
See #303 for more information about this.
